pkg/config/config.go

```diff
@@ -828,6 +828,9 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("runtime_security_config.remote_tagger", true)
 	bindEnvAndSetLogsConfigKeys(config, "runtime_security_config.endpoints.")
 
+	// Serverless Agent
+	config.BindEnvAndSetDefault("serverless_logs_enabled", true)
```

Comment:
I'm very sorry I've made a typo in my previous comment, that would be: `serverless.logs_enabled`...